docs/releases.md

```diff
@@ -1,5 +1,20 @@
 # Release Notes
 
+## 1.1.17
+Fixes part of this release:
+ 1. Console.WriteLine support for .NetCore Projects . [#18](https://github.com/Microsoft/testfx/issues/18) [enhancement]

```

Comment:
you missed this: https://github.com/Microsoft/testfx/issues/23. Its open just because we want to make it the default.